From dbdf3f15e7ed9d5462a6ece09dd1c57ca430d912 Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Tue, 10 Sep 2013 17:10:10 +1000
Subject: [PATCH] config: set "auto_seen" after processing the auto line.

Otherwise when we process an empty autoline (to be sure to
capture the MDADM_CONF_AUTO environment variable) we can end up
setting everything to 'yes' which over-rides 'no'.

Signed-off-by: NeilBrown <neilb@suse.de>
---
 config.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.c b/config.c
index f1a70c5..a35807c 100644
--- a/config.c
+++ b/config.c
@@ -575,6 +575,7 @@ void autoline(char *line)
 
 	if (auto_seen)
 		return;
+	auto_seen = 1;
 
 	/* Parse the 'auto' line creating policy statements for the 'auto' policy.
 	 *
-- 
1.9.3

